### ISSUE TITLE
Scroll up/down while dragging items?

### ISSUE THREAD

**Issue by lars1** [buttonwoodcx/bcx-aurelia-dnd#13]
How could I go about adding "automatic" vertical scrolling while dragging items in a long list that overflows the container? 

Kind of like [this demo on CodePen](https://codepen.io/GreenSock/pen/YPvdYv) (it's not my code, but shows the issue).

This is not a bug. I'm just asking for ideas or pointers on how to do it.

**Comment by 3cp** [comment]
dragula showed me dom-autoscroll bevacqua/dragula#633

dom-autoscroll is fantastic, and it's quite easy to use with bcx-aurelia-dnd or bcx-aurelia-reorderable-repeat.

It's much much better than my previous manual solution. I have created two examples in the doc:

https://buttonwoodcx.github.io/doc-bcx-aurelia-dnd/examples-reorderable-repeat/auto-scroll
https://buttonwoodcx.github.io/doc-bcx-aurelia-dnd/examples/auto-scroll

My old manual scrolling example is removed. BTW I updated the doc site to dumber-gist if you have not heard of it.
